official/utils/arg_parsers/__init__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from .parsers import LocationParser, DeviceParser, SupervisedParser, \
```

Comment:
Let's stick with absolute imports. Both for pep8 and for the sake of rules that convert this code internally.

Reply:
Sure. Fixed.